templates/course_info.html

```diff
@@ -43,42 +43,74 @@ <h2 class="sub-header">Assistant(s) for this year ({{ course.year }} - {{ course
-                         aria-labelledby="assistants-tab">
-                        <table class="table table-hover mt-3">
-                            <thead>
-                            <tr>
-                                <th>Last name</th>
-                                <th>First name</th>
-                            </tr>
-                            </thead>
-                            <tbody>
-                            </tbody>
-                        </table>
-                    </div>
-                    <!-- Evaluations Pane -->
-                    <div class="tab-pane fade show" id="evaluations" role="tabpanel" aria-labelledby="evaluations-tab">
-                        <!-- Content for Evaluations -->
-                        <br>
-                        <p>Content for evaluations will go here.</p>
+                        <!-- Tab panes -->
+                        <div class="tab-content">
+                            <!-- Assistants Pane -->
+                            <div class="tab-pane fade show active" id="assistants-{{ course.year }}" role="tabpanel"
+                                 aria-labelledby="assistants-tab-{{ course.year }}">
+                                <table class="table table-hover mt-3">
+                                    <thead>
+                                    <tr>
+                                        <th>Last name</th>
+                                        <th>First name</th>
+                                    </tr>
+                                    </thead>
+                                    <tbody>
+                                    </tbody>
+                                </table>
+                            </div>
+                            <!-- Evaluations Pane -->
+                            <div class="tab-pane fade show" id="evaluations-{{ course.year }}" role="tabpanel"
+                                 aria-labelledby="evaluations-tab-{{ course.year }}">
+                                <!-- Content for Evaluations -->
+                                <table class="table table-hover" id="evaluationTable">
+                                    <thead>
+                                    <tr>
+                                        <th>Course name</th>
+                                        <th>Year</th>
+                                        <th>User</th>
+                                    </tr>
+                                    </thead>
+                                    <tbody>
+                                    {% if evaluations is none %}
+                                        <tr>
+                                            <td colspan="3">No evaluations for this year.</td>
+                                        </tr>
+                                    {% endif %}
+                                    {% for evaluation in evaluations %}
+                                        {% if evaluation.course_year < course.year %}
```

Comment:
```suggestion
                                        {% if evaluation.course_year == course.year %}
```

Not sure how many times you want to show the History table, but either `==` for each year, either one unique table with no filter at all.

Reply:
This is a history of assessments for each year of a course. So I thought that for a course in the year 2024, all the assessments equal to or preceding that year would be displayed.